### PR TITLE
[WIP] Python 3 compatibility

### PR DESCRIPTION
**Pull request by banilo**
Tries to address #128

### CODE REVIEW

File: examples/coreg_demos.py [hunk 1]
```diff
@@ -4,6 +4,7 @@
 It demos coregistration on a variety of datasets including:
 SPM single-subject auditory, NYU rest, ABIDE, etc.
 """
+from __future__ import print_function
```
Comment (AlexandreAbraham) [inline examples/coreg_demos.py]: You don't need it.

**Comment by dohmatob** [comment]
Any update ? We could discuss the impediments here.

**Comment by banilo** [comment]
The easiest would probably be to make .yml and related parts of the repo as
close as possible to nilearn.


2015-08-19 14:47 GMT+02:00 DOHMATOB Elvis <notifications@github.com>:

> Any update ? We could discuss the impediments here.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/neurospin/pypreprocess/pull/149#issuecomment-132578591>
> .
>



-- 
Viele Grüße,
Danilo

**Comment by dohmatob** [comment]
+1

File: pypreprocess/reporting/hierarchical_reporter.py [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+#         gallery_markup = base_reporter.get_gallery_html_markup().substitute(
+#             thumbnails=thumbnails)
+
+#         html += "<br>%s<br clear='all'>" % gallery_markup
+
+#     html += "<ul>" + "".join(["<li>%s</li>" % trie2html(child) + "</ul>"
+#                               for child in children])
+
+#     return html
+
```
Comment (bthirion) [inline pypreprocess/reporting/hierarchical_reporter.py]: Could this be cleaned ?

**Comment by mrahim** [comment]
For the moment `Nipype` is not compatible with `Python 3`.
Closing.

**Pull request closed without merging**